### PR TITLE
doc/rgw: edit admin.rst - rate limit management

### PR DESCRIPTION
**Pull request by zdover23**
Edit the "Rate Limit Management" section in doc/radosgw/admin.rst.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

**Review by cbodley — APPROVED** [review]

File: doc/radosgw/admin.rst [hunk 1]
```diff
@@ -627,21 +631,26 @@ time has elapsed, "user A" will be able to send ``GET`` requests again.
   user.
 
 - **Maximum Read Ops:** The ``--max-read-ops`` setting allows you to specify
-  the maximum number of read ops per minute per RGW. A 0 value disables this setting (which means unlimited access).
+  the maximum number of read ops per minute per RGW. A ``0`` value disables
```
Comment (anthonyeleven) [inline doc/radosgw/admin.rst:634]: Suggest "RGW instance" or "RGW daemon".

File: doc/radosgw/admin.rst [hunk 2]
```diff
@@ -627,21 +631,26 @@ time has elapsed, "user A" will be able to send ``GET`` requests again.
...
+  this setting (which means unlimited access).
   
-- **Maximum Read Bytes:** The ``--max-read-bytes`` setting allows you to specify
-  the maximum number of read bytes per minute per RGW. A 0 value disables this setting (which means unlimited access).
+- **Maximum Read Bytes:** The ``--max-read-bytes`` setting allows you to
+  specify the maximum number of read bytes per minute per RGW. A ``0`` value
```
Comment (anthonyeleven) [inline doc/radosgw/admin.rst:638]: limit read bytes per minute per RGW instance.  A ``0`` value disables throttling.

File: doc/radosgw/admin.rst [hunk 3]
```diff
@@ -627,21 +631,26 @@ time has elapsed, "user A" will be able to send ``GET`` requests again.
...
+  disables this setting (which means unlimited access).
 
 - **Maximum Write Ops:** The ``--max-write-ops`` setting allows you to specify
-  the maximum number of write ops per minute per RGW. A 0 value disables this setting (which means unlimited access).
+  the maximum number of write ops per minute per RGW. A ``0`` value disables
+  this setting (which means unlimited access).
```
Comment (anthonyeleven) [inline doc/radosgw/admin.rst:643]: congruent with above

File: doc/radosgw/admin.rst [hunk 4]
```diff
@@ -627,21 +631,26 @@ time has elapsed, "user A" will be able to send ``GET`` requests again.
...
   
-- **Maximum Write Bytes:** The ``--max-write-bytes`` setting allows you to specify
-  the maximum number of write bytes per minute per RGW. A 0 value disables this setting (which means unlimited access).
+- **Maximum Write Bytes:** The ``--max-write-bytes`` setting allows you to
+  specify the maximum number of write bytes per minute per RGW. A ``0`` value
```
Comment (anthonyeleven) [inline doc/radosgw/admin.rst:646]: congruent with above

**Review by anthonyeleven — APPROVED** [review]

File: doc/radosgw/admin.rst (hunk 1 above)
Comment (zdover23) [reply doc/radosgw/admin.rst:634]: Accepted.

File: doc/radosgw/admin.rst (hunk 2 above)
Comment (zdover23) [reply doc/radosgw/admin.rst:638]: Accepted.

File: doc/radosgw/admin.rst (hunk 3 above)
Comment (zdover23) [reply doc/radosgw/admin.rst:643]: Accepted.

File: doc/radosgw/admin.rst (hunk 4 above)
Comment (zdover23) [reply doc/radosgw/admin.rst:646]: Accepted.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/55128 - Reef backport
https://github.com/ceph/ceph/pull/55129 - Quincy backport

**Pull request merged**